templates/base.html

```diff
@@ -66,6 +66,11 @@
 		</main>
 		{% endblock %}
 		{% block footer %}
+			<footer>
+				<div>
+					<a href="/instance-info">View instance information</a>
```

Comment:
This isn't bad, but is there any chance we can make this a little prettier? 🙂

Reply:
I tried to spruce it up - nothing too fancy but now it's got some shadows and reacts to the cursor. It'll be up at https://685.fly.dev in a few minutes when it finishes compiling, if you'd like to take a look.